fdbclient/include/fdbclient/Tenant.h

```diff
@@ -114,16 +114,16 @@ struct TenantMetadataSpecification {
 
 struct TenantMetadata {
 private:
-	static inline TenantMetadataSpecification instance = TenantMetadataSpecification("\xff/"_sr);
```

Comment:
Changed to on-demand initialization of the singleton. Otherwise static initialization calls the constructor of TenantMetadataSpecification, which is using currentProtocolVersion. At this point currentProtocolVersion is not yet defined, because we have not yet evaluated server or client options to determine if it has to be set to a future protocol version.